src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -231,4 +264,49 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			}
 		}
 	}
+
+	/**
+	 * Instantiate modal for use as callout when inserting Link or Image into markdown.
+	 * @param calloutStyle Style of callout passed in to determine which callout is rendered.
+	 * Returns markup created after user enters values and submits the callout.
+	 */
+	private async createCallout(type: MarkdownButtonType, triggerElement: HTMLElement): Promise<ILinkCalloutDialogOptions> {
+		const triggerPosX = triggerElement.getBoundingClientRect().left;
+		const triggerPosY = triggerElement.getBoundingClientRect().top;
+		const triggerHeight = triggerElement.offsetHeight;
+		const triggerWidth = triggerElement.offsetWidth;
+		const dialogProperties = { xPos: triggerPosX, yPos: triggerPosY, width: triggerWidth, height: triggerHeight };
+		let calloutOptions;
+		/**
+		 * Width value here reflects designs for Notebook callouts.
+		 */
+		const width: DialogWidth = 452;
```

Comment:
This should be defined in a more common location - and given a name like

`const CALLOUT_WIDTH: DialogWidth = 452;`

Reply:
Sure. I just moved this class to a new location, but happy to move it into `calloutDialog.ts`